modules/publication/jsx/uploadForm.js

```diff
@@ -27,12 +27,8 @@ class PublicationUploadForm extends React.Component {
 
   validateEmail(field, email) {
     let formErrors = this.state.formErrors;
-
-    // don't supply error if email is blank
-    if (email === '' || email === null || email === undefined) {
-      delete formErrors[field];
-      // if email is invalid, set error, else nullify error
-    } else if (/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(email) === false) {
+    const emailDiv = document.getElementById(field);
+    if (!emailDiv.checkValidity()) {
```

Comment:
this doesn't appear to be defined?

Reply:
@driusan I think I modified the previous code to be similar but the element emailDiv should be defined from the 'field' parameter that's passed into validateEmail.

Reply:
I mean the function `checkValidity` isn't defined.

Reply:
@driusan See https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/checkValidity